typescript/infra/scripts/funding/update-balance-threshold-config.ts

```diff
@@ -2,93 +2,96 @@ import { checkbox } from '@inquirer/prompts';
 import yargs from 'yargs';
 
 import { ChainMap } from '@hyperlane-xyz/sdk';
-import {
-  LogFormat,
-  LogLevel,
-  configureRootLogger,
-  rootLogger,
-} from '@hyperlane-xyz/utils';
+import { rootLogger } from '@hyperlane-xyz/utils';
 
-import rawDailyBurn from '../../config/environments/mainnet3/balances/dailyBurn.json';
+import rawDailyBurn from '../../config/environments/mainnet3/balances/dailyRelayerBurn.json';
 import { readJSONAtPath, writeJsonAtPath } from '../../src/utils/utils.js';
 import { withBalanceThresholdConfig } from '../agent-utils.js';
 
 import {
-  BalanceThresholdConfig,
-  configFileNameMapping,
-  dailyBurnMultiplier,
+  BalanceThresholdType,
+  balanceThresholdConfigMapping,
 } from './utils/constants.js';
-import { THRESHOLD_CONFIG_PATH } from './utils/grafana.js';
-import { formatDailyBurn } from './utils/utils.js';
+import {
+  THRESHOLD_CONFIG_PATH,
+  formatDailyRelayerBurn,
+  orderThresholds,
+} from './utils/grafana.js';
 
 const dailyBurn: ChainMap<number> = rawDailyBurn;
 
-export const configChoiceLabelMapping: Record<BalanceThresholdConfig, string> =
-  {
-    [BalanceThresholdConfig.RelayerBalance]: 'Desired Relayer Balance',
-    [BalanceThresholdConfig.LowUrgencyKeyFunderBalance]:
-      'Low Urgency Key Funder Balance Alert Threshold',
-    [BalanceThresholdConfig.LowUrgencyEngKeyFunderBalance]:
-      'Low Urgency Eng Key Funder Balance Alert Threshold',
-    [BalanceThresholdConfig.HighUrgencyRelayerBalance]:
-      'High Urgency Relayer Balance',
-  };
+const exclusionList = ['osmosis'];
 
 async function main() {
-  configureRootLogger(LogFormat.Pretty, LogLevel.Info);
   const { balanceThresholdConfig } = await withBalanceThresholdConfig(
     yargs(process.argv.slice(2)),
   ).argv;
 
-  const configToUpdate: BalanceThresholdConfig[] = balanceThresholdConfig
+  const configToUpdate: BalanceThresholdType[] = balanceThresholdConfig
     ? [balanceThresholdConfig]
-    : [];
-
-  if (configToUpdate.length === 0) {
-    const selectedConfig = await checkbox({
-      message: 'Select the balance threshold config to update',
-      choices: Object.values(BalanceThresholdConfig).map((config) => ({
-        name: configChoiceLabelMapping[config],
-        value: config,
-        checked: true, // default to all checked
-      })),
-    });
-
-    configToUpdate.push(...selectedConfig);
-  }
+    : await checkbox({
+        message: 'Select the balance threshold config to update',
+        choices: Object.values(BalanceThresholdType).map((config) => ({
+          name: balanceThresholdConfigMapping[config].choiceLabel,
+          value: config,
+          checked: true, // default to all checked
+        })),
+      });
```

Comment:
can i also get a cli option that skips the prompt entirely and goes straight to the updates? something like `--yes` or `--all`